manifests/j/Jeskola/Buzz/1503/Jeskola.Buzz.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Jeskola.Buzz
+PackageVersion: 1503

```

Comment:
The Version is blank in ARP (Programs and Features) could you contact the dev about this?
This is requires for `winget upgrade` to work.